types/yandex-maps/yandex-maps-tests.ts

```diff
@@ -175,3 +175,28 @@ ymaps.suggest('Mos', {
 }).then(items => {
     return items.filter(el => el.value.toLowerCase() === 'moscow');
 }).then(console.log);
+
+const geocodeProvider: ymaps.IGeocodeProvider = {
+    suggest: (
+        _request: string,
+        _options?:
+            | { boundedBy?: number[][] | undefined; results?: number | undefined; strictBounds?: boolean | undefined },
+    ): Promise<object> => {
+        throw new Error('Function not implemented.');
+    },
+    geocode: (
+        _request: string | ReadonlyArray<number>,
+        _options?:
+            | {
+                  boundedBy?: number[][] | undefined;
+                  results?: number | undefined;
+                  skip?: number | undefined;
+                  strictBounds?: boolean | undefined;
+              },
+    ): Promise<object> => {
+        throw new Error('Function not implemented.');
+    },
```

Comment:
Why do you need explicitly declare types of these methods? You've already set type for a `geocodeProvider` const.

Reply:
Fixed.